examples/drivers/windows/otLwf/filter.h

```diff
@@ -144,7 +144,7 @@ typedef struct _MS_FILTER
     IN6_ADDR                    otLinkLocalAddr;
     otNetifAddress              otAutoAddresses[OPENTHREAD_CONFIG_NUM_SLAAC_ADDRESSES];
 #if OPENTHREAD_ENABLE_DHCP6_CLIENT
-    otNetifAddress              otDhcpAddresses[OPENTHREAD_CONFIG_NUM_DHCP_PREFIXES];
+    otDhcpAddress               otDhcpAddresses[OPENTHREAD_CONFIG_NUM_DHCP_PREFIXES];
```

Comment:
I believe there are going to be a number of changes you need to make in `examples/drivers/windows/otLwf/address.c` now.